addon/mixins/recognizers.js

```diff
@@ -29,6 +37,8 @@ export default Mixin.create({
   },
 
   __setupRecognizers: on('didInsertElement', function() {
+    if (this.get('__fastboot.isFastBoot')) { return; }
```

Comment:
Perhaps my question is related to this code path. Fastboot will not call this hook so... Curios I am if someone had an error that provoked this PR.  

Thanks for hearing me out.